app/domain/audits/find_content.rb

```diff
@@ -1,15 +1,23 @@
 module Audits
   class FindContent
-    def self.call(filter = Filter.new)
-      scope = Content::Query.new
-                .page(filter.page)
-                .organisations(filter.organisations, filter.primary_org_only)
-                .document_type(filter.document_type)
-                .after(filter.after)
-                .theme(filter.theme_id)
-                .scope
+    def self.paged(filter)
+      scope = query_from_filter(filter).content_items
+      filter.audited_policy.call(scope)
+    end
 
+    def self.all(filter)
+      scope = query_from_filter(filter).all_content_items
       filter.audited_policy.call(scope)
     end
+
+    def self.query_from_filter(filter)
```

Comment:
What about renaming it to: 
```ruby
def self.query(filter)
```

`from_filter` is implicit in the parameter name.